pkg/network/port.go

```diff
@@ -31,7 +31,7 @@ var statusMap = map[ConnectionType]int64{
 }
 
 // ReadInitialState reads the /proc filesystem and determines which ports are being listened on
-func ReadInitialState(procRoot string, protocol ConnectionType, collectIPv6 bool, readNS bool) (map[PortMapping]uint32, error) {
+func ReadInitialState(procRoot string, protocol ConnectionType, collectIPv6 bool) (map[PortMapping]uint32, error) {
```

Comment:
please refactor in separate PRs when possible to avoid cluttering PRs.

Reply:
This is a side affect of using the net ns for udp namespaces.